back-end/src/lms/functions.py

```diff
@@ -21,3 +26,23 @@ def get_date_range(date_from: datetime, date_to: datetime,
         cur_date += timedelta(7)
 
     return dates
+
+
+def search_persons(model: Model, serializer: Serializer,
+                   field: str) -> list[dict]:
+    return serializer(model.objects.filter(
+        Q(name__icontains=field) | Q(surname__icontains=field) |
+        Q(patronymic__icontains=field)),
+                      many=True)
+
+
+def persons_response(data: list[dict]) -> list[dict]:
+    result = []
+    for item in data.data:
+        if item.get('user', None):
+            res = {
+                'id': item['user'],
+                'fullname': item['fullname']
+            }
+            result.append(res)
+    return result
```

Comment:
I would move those functions to `back-end/src/lms/views/persons.py` as these are functions specific to the `Persons` API